views/challenge/create.php

```diff
@@ -74,29 +76,42 @@
                 document.getElementById("challenge-label_exchange_rate").style.visibility = "visible"; 
                 document.getElementById("challenge-exchange_rate").style.visibility = "visible"; 
                 document.getElementById("challenge-specific_reward_asset_id").parentElement.style.visibility = "visible";
+                document.getElementById("challenge-label_specific_reward_asset").style.visibility = "visible"; 
+
                  '
             ]);
             ?>
-            <label class ="control-label "for="challenge-exchange_rate" style="visibility:hidden" id ="challenge-label_exchange_rate">Exchange rate </label>
-            <?= $form->field($model, 'exchange_rate')->textInput(['maxlength' => true, 'style' => 'visibility:hidden'
-            ])->label(false) ?>
-
-            <?=
-            $form->field($model, 'specific_reward_asset_id')->widget(Select2::class, [
```

Comment:
why not wrapping the form field in `<div class="col-md-4">` for example ? this way maybe you will not need to each time hide the label and the input rather you will hide the wrapper div ?